ruby/ql/lib/codeql/ruby/frameworks/ActionView.qll

```diff
@@ -216,4 +216,138 @@ class FileSystemResolverAccess extends DataFlow::CallNode, FileSystemAccess::Ran
 
   override DataFlow::Node getAPathArgument() { result = this.getArgument(0) }
 }
+
 // TODO: model flow in/out of template files properly,
+// TODO: Move the classes and predicates above inside this module.
+/** Modeling for `ActionView`. */
+module ActionView {
+  /**
+   * Action view helper methods which are XSS sinks.
+   */
+  module Helpers {
+    /**
+     * A call to ActionView helpers which renders its argument without escaping.
+     * The argument should be treated as an XSS sink. In the documentation for
+     * classes in this module, the vulnerable argument is named `x`.
+     */
+    abstract class RawHelperCall extends MethodCall {
+      /**
+       * Get an argument which is rendered without escaping.
```

Comment:
The [style guide](https://github.com/github/codeql/blob/main/docs/qldoc-style-guide.md#predicates-with-result) suggests:
```suggestion
       * Gets an argument which is rendered without escaping.
```